src/components/tx/TxData.js

```diff
@@ -493,6 +493,29 @@ class TxData extends React.Component {
       return <ul>{v}</ul>;
     };
 
+    const renderNewUiListWithLinks = hashes => {
```

Comment:
chore: This function is called 9 times, and in each call the interface has to decide whether to render the new UI or the old one.

The function by the interface should continue to be `renderListWithLinks()` and, if this function detects the new UI is active, it returns this `renderNewUiListWithLinks()` function instead. This improves centralization and code clarity.

Reply:
Thanks for the suggestion, it has been corrected for clarity. Waiting for review.🙂